doc-QuickStart_OpenShift_Provider_Guide/topics/image-scanning.adoc

```diff
@@ -0,0 +1,80 @@
+== Configuring Image Scanning
+
+{product-title} manages vulnerability scanning of container images. When an OpenShift provider is added, OpenShift images from the internal registry are discovered. To enable image scanning, perform the following configuration steps:
+
+. Navigate to menu:Compute[Containers > Providers].
+. Select the *OpenShift* provider checkbox.
```

Comment:
For this step, I was a little confused when running through the procedure on my test instance. Would it be OK to reword as something like "Select the OpenShift providers to enable scanning on." or "Select the checkboxes of the OpenShift providers to scan." or similar?